vars/main.yml

```diff
@@ -1 +1,4 @@
-timesync_ntp_provider_os_default: "{{ 'ntp' if ansible_distribution in ['RedHat', 'CentOS'] and ansible_distribution_version is version('7.0', '<') else 'chrony' }}"
+timesync_ntp_provider_os_default: "{{
+    'ntp' if ansible_distribution in ['RedHat', 'CentOS'] and
+        ansible_distribution_version is version('7.0', '<')
```

Comment:
I think it would be better to move this variable to the distribution-specific vars files instead of this conditional.